HomeWork3/DemoHomeWork3.java

```diff
@@ -0,0 +1,75 @@
+import inheritance.people.Person;
+import task1.impl.PersonListImpl;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.io.Console;
+
+public class DemoHomeWork3 {
+
+    public static void setParam(int param1, int param2){
+
+    }
+
```

Comment:
why do you need the empty method?

Reply:
Sorry, it's copy-paste error. It will be deleted